fast-af/fast-checkout-woocommerce
# PINT-387 Add Fast button Gutenberg blocks

# Description

This update adds [Gutenberg ](https://wordpress.org/gutenberg/)blocks for displaying the Fast Checkout and Fast Login buttons. Gutenberg is the visual editor for WordPress that replaces the previous WYSIWYG editor.

# Testing instructions

1. Login to the [WordPress admin on the staging site](https://fasttestdev.wpcomstaging.com/wp-admin/).
2. Create a new page or post.
3. Add the Fast Checkout Product Button, Fast Checkout Cart Button, and Fast Login Button blocks to the page.
4. Verify that the buttons are rendered properly on the page/post when appropriate.

# Checks
